docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Server Access Controls
+description: Role-based access control (RBAC) for Teleport Server Access.
+---
+
+# Server Access Role-Based Access Control
+
+Role-based access control (or RBAC, for short) allows administrators to set up
+granular access policies for servers connected to Teleport.
+
+An example of a policy could be, *"server administrators have access to
+everything, QA team and engineers have full access to staging servers, and
+engineers can gain temporary access to the production server in case of
+emergency"*.
+
+For a more general description of Teleport roles and examples see [RBAC](../access-controls/introduction.mdx), as
+this section focuses on configuring RBAC for Server Access.
+
+## Role configuration
+
+Teleport's "role" resource provides the following instruments for restricting
+server access:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: developer
+spec:
+  # SSH options used for user sessions
+  options:
+    # max_session_ttl defines the TTL (time to live) of SSH certificates
+    # issued to the users with this role.
+    max_session_ttl: 30h0m0s
+    # forward_agent controls whether SSH agent forwarding is allowed
+    forward_agent: true
+
+    # port_forwarding controls whether TCP port forwarding is allowed
+    port_forwarding: true
+
+  allow:
+    # Label selectors for server instances this role has access to.
+    #
+    # These will be matched against the static/dynamic labels set on the
+    # ssh service.
+    node_labels:
+      environment: ["dev", "stage"]
+
+    # logins array defines the OS logins a user is allowed to use.
+    # A few special variables are supported here (see below)
+    logins: ["ubuntu", "debian"]
+
+  deny:
+    logins: ["root"]
+```
+
+<Admonition
+  type="note"
+  title="Deny Rules"
+>
+  Deny rules will match greedily. In the example above, any attempt to use `root`
```

Comment:
If we're going to explain the allow/deny rule logic here, I think we need a more complete explanation. Right now, there is little indication of how each field in a role works together when a user attempts to access a Node.

We can use the `node-logins.mdx` partial added in [this PR](https://github.com/gravitational/teleport/pull/12583/files) for this, though we would need to wait for that PR to be merged.